modules/serverless-build-events-kn.adoc

```diff
@@ -0,0 +1,75 @@
+[id="serverless-build-events-kn_{context}"]
+= Building events
+
+You can use the builder-like interface of the `kn event build` command to build an event. You can then send that event at a later time or use it in another context.
+
+.Procedure
+
+* Build an event:
++
+[source,terminal]
+----
+$ kn event build --field <field-name>=<value> --type <type-name> --id <id> --output <format>
+----
+** The `--field` option adds data to the event as a field-value pair. You can use it multiple times.
+** The `--type` option specifies the type of the event.
+** The `--id` option specifies the ID of the event.
+** You can use the `json` or `yaml` arguments with the `--output` option to change the output format of the event.
```

Comment:
For a list of annotations, we usually use annotation tags (`<1>` etc). For this one, that would also involve breaking the command into multiple lines. Alternatively, you could add a line that just says `where:` above line 14.